src/main/java/com/mraof/minestuck/entity/consort/EnumConsort.java

```diff
@@ -13,10 +13,10 @@
 
 public enum EnumConsort	//TODO Could ideally be changed into a registry.
 {
-	SALAMANDER(() -> MSEntityTypes.SALAMANDER, "salamander", ChatFormatting.YELLOW, () -> MSSoundEvents.ENTITY_SALAMANDER_AMBIENT, () -> MSSoundEvents.ENTITY_SALAMANDER_HURT, () -> MSSoundEvents.ENTITY_SALAMANDER_DEATH),
-	TURTLE(() -> MSEntityTypes.TURTLE, "turtle", ChatFormatting.LIGHT_PURPLE, () -> null, () -> MSSoundEvents.ENTITY_TURTLE_HURT, () -> MSSoundEvents.ENTITY_TURTLE_DEATH),
-	NAKAGATOR(() -> MSEntityTypes.NAKAGATOR, "nakagator", ChatFormatting.RED, () -> MSSoundEvents.ENTITY_NAKAGATOR_AMBIENT, () -> MSSoundEvents.ENTITY_NAKAGATOR_HURT, () -> MSSoundEvents.ENTITY_NAKAGATOR_DEATH),
-	IGUANA(() -> MSEntityTypes.IGUANA, "iguana", ChatFormatting.AQUA, () -> MSSoundEvents.ENTITY_IGUANA_AMBIENT, () -> MSSoundEvents.ENTITY_IGUANA_HURT, () -> MSSoundEvents.ENTITY_IGUANA_DEATH);
+	SALAMANDER(MSEntityTypes.SALAMANDER::get, "salamander", ChatFormatting.YELLOW, () -> MSSoundEvents.ENTITY_SALAMANDER_AMBIENT, () -> MSSoundEvents.ENTITY_SALAMANDER_HURT, () -> MSSoundEvents.ENTITY_SALAMANDER_DEATH),
```

Comment:
`RegistryObject<T>` implements `Supplier<T>`, so you should be able to use the registry object directly.